doc/api/buffer.md

```diff
@@ -1118,6 +1118,12 @@ const { Buffer } = require('node:buffer');
 const buf = Buffer.from([0x62, 0x75, 0x66, 0x66, 0x65, 0x72]);
 ```
 
+If `array` is an `Array`-like object (that is, one with a `length` property of
+type `number`), it is treated as if it is an array, unless it is a `Buffer` or a
+`Uint8Array`. This means all other `TypedArray` variants get treated as an
+`Array`. To create a `Buffer` from the underlying `ArrayBuffer`, use the
+`.buffer` property directly.
```

Comment:
> To create a `Buffer` from the underlying `ArrayBuffer`, use the `.buffer` property directly.

This sentence might imply `Buffer.from(src.buffer)` which is a common mistake ignoring `offset` of source view.
How about either adding an example(`Buffer.from(src.buffer, src.offset, src.byteLength)`) or suggesting [`Buffer.copyBytesFrom(view[, offset[, length]])`](https://nodejs.org/api/buffer.html#static-method-buffercopybytesfromview-offset-length)?

Reply:
I think `Buffer.copyBytesFrom` is the most correct suggestion here, so I'll go with that 👍. I'll make that change now.

Reply:
@LiviaMedeiros Changed in latest push. PTAL.